src/apps/service/apps/combined.clj

```diff
@@ -57,7 +57,7 @@
          (util/combine-app-listings params)))
 
   (adminSearchApps [_ search-term params]
-    (->> (map #(.adminSearchApps % search-term (select-keys params [:search :app-subset])) clients)
+    (->> (map #(.adminSearchApps % search-term (select-keys params [:search :app-subset :start_date :end_date])) clients)
```

Comment:
It looks like these query params will be passed to Agave app listings, so we should make sure Agave app listings will still work with these start and end date params, or make sure they are filtered out of the listing query.

Also, Agave apps with their job stats are included in this admin listing endpoint:
https://github.com/cyverse-de/apps/blob/2394f162b591f5edb4411542794c183d3cc872fc/src/apps/service/apps/agave/listings.clj#L11-L21
So do we also want to filter Agave job stats by these dates?

Reply:
It looks like these extra params won't interfere with Agave app listings:
https://github.com/cyverse-de/mescal/blob/dcd101c7b6d6066c386a4de0729af6cf363663b3/src/mescal/agave_v2.clj#L97-L99

But I think we probably want to also filter Agave app stats with these start/end date params.

Reply:
Sounds good. I will add date filtering to agave stats and option for sending either `start_date` or `end_date`. I still don't understand why my Intellij / clojure cursive won't format code properly.